api_tests/nodes/views/test_node_draft_registration_detail.py

```diff
@@ -156,9 +156,10 @@ def draft_registration(self, user, project_public, schema):
 
     @pytest.fixture()
     def reg_schema(self):
-        return RegistrationSchema.objects.get(
-            name='OSF Preregistration',
-            schema_version=SCHEMA_VERSION)
+        schema = RegistrationSchema.objects.get_latest_version(name='OSF Preregistration')
+        schema.active = True
+        schema.save()
+        return schema
```

Comment:
Might be caused by schema / fixture relying on old test DB before upgrade while new test DB is slight different.
The fix is fine here since it serves the purpose. However, we might need to update the tests if the `latest_schema` gets updated. Not a blocker but pinning it to a version will help.